docs/container-images.md

```diff
@@ -188,19 +213,271 @@ use. There are some requirements on this image though:
    where your code is going to run in the cloud
 3. The entrypoint must be set to a script which executes
    `/usr/bin/python3 -m sematic.resolvers.worker "$@"`
-4. `/usr/bin/python3` must be a valid python interpreter in the image
+4. `/usr/bin/python3` must be a valid Python interpreter in the image
 5. The home directory inside the image must be writable
 6. The `/sematic/bin/` directory inside the image must be writable
 7. `/sematic/bin` must be on your `PATH`
 
-## Working with images
+#### Multiple base images with Bazel
+
+Please see [Support for multiple images](./multiple-base-images.md).
+
+### Docker
```

Comment:
We might want to move the `Docker` section above the `Bazel` section, since I suspect native Docker builds will be more common for most users.

Reply:
+1